deploy/chart/templates/_helpers.tpl

```diff
@@ -30,6 +30,15 @@ Create chart name and version as used by the chart label.
 {{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" }}
 {{- end }}
 
+{{/*
+Namespace for all resources to be installed into
+If not defined in values file then the helm release namespace is used
+By default this is not set so the helm release namespace will be used
```

Comment:
I think it's worth explaining why this exists. Is that because of https://github.com/helm/helm/issues/5358? If it is, could you add a note, e.g.

```suggestion
By default this is not set so the helm release namespace will be used.


This gets around an problem within Helm discussed in
https://github.com/helm/helm/issues/5358.
```